db/schema.rb

```diff
@@ -20,6 +20,7 @@
   create_table "users", force: :cascade do |t|
     t.string   "name",                   limit: 255,              null: false
     t.integer  "role",                   default: 0,  null: false
+    t.text     "profile_photo"
```

Comment:
It's carrier wave uploader, it requires a text field.  I'm not using the attachment framework for icons.